src/python/pants/testutil/engine/util.py

```diff
@@ -22,7 +22,7 @@
 
 
 @dataclass(frozen=True)
-class MockedYieldGet:
+class MockGet:
```

Comment:
This also needs to be updated in the docstring :smile:

Reply:
Good catch. Thank you!